File: app/src/main/java/com/stevenschoen/putionew/model/responses/FileResponse.java
package com.stevenschoen.putionew.model.responses;

import com.stevenschoen.putionew.model.ResponseOrError;
import com.stevenschoen.putionew.model.files.PutioFile;

public class FileResponse extends ResponseOrError.BasePutioResponse {
	private PutioFile file;

	public PutioFile getFile() {
		return file;
	}
}

File: app/src/main/java/com/stevenschoen/putionew/model/responses/FileResponse.kt
package com.stevenschoen.putionew.model.responses

import com.stevenschoen.putionew.model.ResponseOrError
import com.stevenschoen.putionew.model.files.PutioFile

class FileResponse(
    val file: PutioFile
) : ResponseOrError.BasePutioResponse()
